src/routes/SheltersPege/SheltersPege.js

```diff
@@ -77,34 +100,210 @@ export default class SheltersPege extends Component {
+      <>
+        {stateOptions}
+      </>
+    )
+  }
+
+  renderEditShelter() {
+    const {
+      shelterName,
+      address,
+      city,
+      state,
+      zip,
+      email,
+      phoneNumber,
+      website
+    } = this.state;
+
+    return (
+      <div>
+        <form>
+          <div>
+            <table className='shelterTable'>
+              <tbody>
+                <tr>
+                  <th>Shelter Name</th>
+                  <td><Input type='text' className='form-control' required value={shelterName}
+                    onChange={e => this.inputChanged('shelterName', e.target.value)} /></td>
+                </tr>
+                <tr>
+                  <th>Address</th>
+                  <td><Input type='text' className='form-control' required value={address}
+                    onChange={e => this.inputChanged('address', e.target.value)} /></td>
+                </tr>
+                <tr>
+                  <th>City</th>
+                  <td><Input type='text' className='form-control' required value={city}
+                    onChange={e => this.inputChanged('city', e.target.value)} /></td>
+                </tr>
+                <tr>
+                  <th>State</th>
+                  <td>
+                    <select className='form-control' value={state} onChange={e => {
+                      this.setState({ 'state': e.target.value });
+                    }}>
+                      {this.renderUSStates()}
+                    </select>
+                  </td>
+                </tr>
+                <tr>
+                  <th>Zip Code</th>
+                  <td><Input type='number' min='0' max='99999' className='form-control' required value={zip}
+                    onChange={e => this.inputChanged('zip', e.target.value)} /></td>
+                </tr>
+                <tr>
+                  <th>Email</th>
+                  <td><Input type='email' className='form-control' required value={email}
+                    onChange={e => this.inputChanged('email', e.target.value)} /></td>
+                </tr>
+                <tr>
+                  <th>Phone number</th>
+                  <td><Input type='number' className='form-control' required value={phoneNumber}
+                    onChange={e => this.inputChanged('phoneNumber', e.target.value)} /></td>
+                </tr>
+                <tr>
+                  <th>Website</th>
+                  <td><Input type='text' className='form-control' value={website}
+                    onChange={e => this.inputChanged('website', e.target.value)} /></td>
+                </tr>
+              </tbody>
+            </table>
+            <hr />
+          </div>
+          <div className="row">
+            <div className="col-sm-12 text-center">
+              <button className="btn btn-primary btn-md m-2" onClick={this.onClickUpdate}>Update</button>
+              <button className="btn btn-success  btn-md m-2" onClick={this.onClickEdit}>Cancel</button>
```

Comment:
I think this should be changed to `className="btn btn-outline-primary"`

Reply:
much prettier! thank you!